forecast/Sample-Usage.ipynb

```diff
@@ -10,327 +10,69 @@
     "import numpy as np\n",
```

Comment:
I see you split out the sin wave example; what do you want people to take away from reading this ipynb?